src/test/test_bitcoin.cpp

```diff
@@ -25,7 +25,7 @@
 
 CClientUIInterface uiInterface; // Declared but not defined in ui_interface.h
 CWallet* pwalletMain;
-libzerocash::ZerocashParams *pzerocashParams;
+ZCJoinSplit *pzcashParams;
```

Comment:
I'm confused, @ebfull says above that this is the same variable but the variable is defined in `init.cpp` so if we're not linking against that then what this line does is re-define it? C++ is so confusing.